base/Dockerfile

```diff
@@ -6,5 +6,14 @@ MAINTAINER Rob Loach <robloach@gmail.com>
 RUN apt-get update && apt-get install -y mysql-client
 RUN docker-php-ext-install pdo_mysql
 
+# Add the Redis PHP module.
+RUN git clone --branch="php7" https://github.com/phpredis/phpredis.git /tmp/phpredis && cd /tmp/phpredis && \
```

Comment:
@RobLoach is there a particular reason why we aren't using the built in `docker-php-ext-install`? I *think* that is the preferred way to do things

```bash
git clone -b php7 https://github.com/phpredis/phpredis.git /usr/src/php/ext/redis
docker-php-ext-install redis
```

See: http://stackoverflow.com/questions/31369867/how-to-install-php-redis-extension-using-the-official-php-docker-image-approach